templates/cluster-template-crs-cni.yaml

```diff
@@ -193,16 +194,17 @@ spec:
       apt-get remove -y docker docker-engine containerd runc
       apt-get install -y apt-transport-https ca-certificates curl gnupg lsb-release linux-generic jq
       major_vers=$(lsb_release -r | awk '{ print $2 }' | cut -d. -f1)
-      if [[ "$major_vers" -ge 20 ]]; then
+      if [ "$major_vers" -ge 20 ]; then
```

Comment:
this is consistent with elsewhere in this block: https://github.com/kubernetes-sigs/cluster-api-provider-packet/blob/0fe9137c8a0c927c7b97dd310a58fe1b2bdee6a6/templates/cluster-template-crs-cni.yaml#L168